backend/src/test/java/gov/cdc/usds/simplereport/test_util/TestDataFactory.java

```diff
@@ -63,7 +63,7 @@ public Organization createValidOrg() {
   }
 
   public Facility createValidFacility(Organization org) {
-    return createValidFacility(org, "Injection Site");
+    return createValidFacility(org, "Imaginary Site");
```

Comment:
changed this so it doesn't conflict with config facilities if created as part of config org